compatibility-verifier/compCheck.sh

```diff
@@ -169,6 +182,11 @@ COMPAT_TESTER_PATH="pinot-integration-tests/target/pinot-integration-tests-pkg/b
 # create subdirectories for given commits
 workingDir=$1
 testSuiteDir=$(absPath "$2")
+
+brokerConfigFile=${testSuiteDir}/config/BrokerConfig.conf
```

Comment:
Can you set up front with `BROKER_CONF="BrokerConfig.conf"` and then use `${BROKER_CONF}` everywhere? Similarly the other two?

Thanks.